pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -20,64 +21,16 @@ import (
 	"github.com/openshift/insights-operator/pkg/gatherers"
 	"github.com/openshift/insights-operator/pkg/gatherers/common"
 	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/utils"
 )
 
+const gatheringRulesEndpoint = "http://localhost:8080/rules.json"
```

Comment:
It would be good to read this value from config.

Reply:
Good point